src/operator/operator_tune.cc

```diff
@@ -125,16 +125,16 @@ struct static_init_var {
  * \brief Implement tuning objects for a backward unary kernel operator
  */
 #define _IMPLEMENT_UNARY_WORKLOAD_BWD(__op$, __typ$) \
```

Comment:
can we simplify these macros a little? maybe by wrapping all static variables in a struct?

Reply:
Not sure I follow you.

Reply:
There's two static variables that are used externally, tuned_op<>::workload_ (size_t) and tuned_op<>::workload_ex (vector). Those reside in the same struct (tuned_op).  

The other static var is a trivial target for a startup init call and has no use outside of the scope of this file, so it is not included in the tuned_op<> struct.  The call for this variable varies depending on the type of operator (unary, binar, etc).

Other definitions are UseOMP() implementations which vary based upon the type of operator (unary, binary, custom, etc).   For more custom ops, they would be created by hand (like broadcast was).